contracts/switchboard/default-switchboards/FastSwitchboard.sol

```diff
@@ -180,4 +182,99 @@ contract FastSwitchboard is SwitchboardBase {
 
         totalWatchers[srcChainSlug_]--;
     }
+
+    function isNonWatcherRole(bytes32 role_) public pure returns (bool) {
+        if (
+            role_ == TRIP_ROLE ||
+            role_ == UNTRIP_ROLE ||
+            role_ == WITHDRAW_ROLE ||
+            role_ == RESCUE_ROLE ||
+            role_ == GOVERNANCE_ROLE ||
+            role_ == GAS_LIMIT_UPDATER_ROLE
+        ) return true;
+
+        return false;
+    }
+
+    function grantRole(
+        bytes32 role_,
+        address grantee_
+    ) external override onlyOwner {
+        if (isNonWatcherRole(role_)) {
+            _grantRole(role_, grantee_);
+        } else {
+            revert InvalidRole();
+        }
+    }
```

Comment:
Add comment saying why we need to check for watcher role

Reply:
done